src/actions/index.js

```diff
@@ -0,0 +1,16 @@
+
+const ADD_BOOK = 'addBook';
+const DEL_BOOK = 'delBook';
```

Comment:
I'm curious about these line 2 and 3, do you really need them in here?
And CREATE_BOOK action objects seems not completed.

example:
```
const addBook = book => ({
  type: 'CREATE_BOOK',
  id: book.id,
  title: book.title,
  category: book.category,
});


const deleteBook = book => ({
  type: 'DELETE_BOOK',
  id: book,
});
```

Reply:
We are using destructuring object here. 
`const CREATE_BOOK = book => ({
  type: ADD_BOOK,
  book
});`

This is the same as above: 
`const CREATE_BOOK = book => ({
  type: ADD_BOOK,
  book: {
     bookId: 1,
     title: 'Name of book',
     category: 'Action'
  },
});`

I don't know if this answers your doubts. 😄